### PR TITLE
make DNS1123 validation more verbose

### PR DESCRIPTION
**Pull request by suomiy**
bug: https://bugzilla.redhat.com/show_bug.cgi?id=1710901

### CODE REVIEW

File: src/utils/grammar.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+};
+
+export const makeSentence = (sentence, capitalize = true) => {
+  const result = capitalize ? _.capitalize(sentence) : sentence;
+  return !result || result.charAt(result.length) === '.' ? result : `${result}.`;
+};
+
+export const addMissingSubject = (sentence, subject) => {
+  const c = sentence ? sentence.charAt(0) : '';
+  return c.toLowerCase() !== c ? sentence : `${_.capitalize(subject)} ${sentence}`;
```
Comment (mareklibra) [inline src/utils/grammar.js]: Vivat future i18n effort :-(

Comment (suomiy) [reply src/utils/grammar.js]: This feature is already implemented in both fork and `openshift/console`. It just doesn't have its own function. 

i18n will be resolved once its time comes

**Comment by mareklibra** [comment]
Needs rebase, otherwise lgtm

**Comment by suomiy** [comment]
rebased

**Pull request merged**